examples/routeguide-tutorial.md

```diff
@@ -273,6 +273,7 @@ use std::pin::Pin;
 use std::sync::Arc;
 use tokio::sync::mpsc;
 use tonic::{Request, Response, Status};
+use tokio_stream::wrappers::ReceiverStream;
```

Comment:
I think this will require adding `tokio-stream` as a dependency [here](https://github.com/hyperium/tonic/blob/2e88100179f2197121d207cb55c3d105d695d938/examples/routeguide-tutorial.md#generating-client-and-server-code).